Submodels/Parameterizations.py

```
@@ -253,14 +310,27 @@ def GetLeadingEdgeRadius(params: tuple[float, float, float],
+            "y_t": y_t,
+            "x_c": x_c,
+            "y_c": y_c,
+            "z_TE": z_TE,
+            "dz_TE": dz_TE,
+            "r_LE": r_LE,
+            "leading_edge_direction": leading_edge_direction,
+            "trailing_wedge_angle": trailing_wedge_angle,
+            "trailing_camberline_angle": trailing_camberline_angle,
+        }
+
+        return output
```

Ruff (0.8.2)

Unnecessary assignment to `output` before `return` statement

Remove unnecessary assignment

(RET504)